modules/psap-special-resource-operator-installing.adoc

```diff
@@ -0,0 +1,171 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/psap-special-resource-operator.adoc
+
+[id="installing-the-special-resource-operator_{context}"]
+= Installing the Special Resource Operator
+
+As a cluster administrator, you can install the Special Resource Operator (SRO) using the {product-title} CLI or the web console.
+
+[id="install-operator-cli_{context}"]
+== Installing the Special Resource Operator using the CLI
+
+As a cluster administrator, you can install SRO using the CLI.
+
+.Prerequisites
+
+* An {product-title} cluster
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
```

Comment:
Combine bullets 1 and 3? 
* Log in to your {product-title} cluster as a user with `cluster-admin` privileges.

Reply:
Thanks, this should now be fixed, and I also applied this to the other prerequisites in the rest of the SRO pages.